src/perform_step/exponential_rk_perform_step.jl

```diff
@@ -258,19 +259,20 @@ function perform_step!(integrator, cache::ETDRK3Cache, repeat_step=false)
   halfdt = dt/2
   if alg.krylov
     w1_half, w1, w2, w3, phiv_caches = KsCache
+    kwargs = [(:m, min(alg.m, size(A,1))), (:norm, integrator.opts.internalnorm), (:iop, alg.iop), (:cache, tmp)]
```

Comment:
this will need to be a named tuple on v0.7 but that's fine.

Reply:
I'll add a comment for this.